gateway/tests/gateway_test.rs

```diff
@@ -1330,9 +1331,16 @@ fn test_submit_cron_checking_errors() {
     assert_eq!(r.unwrap_err().msg(), "caller not validator");
 }
 
-fn get_epoch_submissions(rt: &mut MockRuntime, epoch: ChainEpoch) -> Option<CronSubmission> {
+fn get_epoch_submissions(
+    rt: &mut MockRuntime,
+    epoch: ChainEpoch,
+) -> Option<EpochVoteSubmissions<CronCheckpoint>> {
     let st: State = rt.get_state();
-    let hamt = st.cron_submissions.load(rt.store()).unwrap();
+    let hamt = st
+        .cron_checkpoint_voting
+        .epoch_vote_submissions()
+        .load(rt.store())
+        .unwrap();
     let bytes_key = BytesKey::from(epoch.to_be_bytes().as_slice());
```

Comment:
Change the epoch key to use this: https://github.com/consensus-shipyard/ipc-actors/blob/dc3eaee617cc6b4d3bd080c52ff82bb730b08cfd/sdk/src/lib.rs#L30

Reply:
updated